src/schema/v2/me/newWorksFromGalleriesYouFollow.ts

```diff
@@ -0,0 +1,46 @@
+import { GraphQLFieldConfig } from "graphql"
+import { connectionFromArraySlice } from "graphql-relay"
+import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+import { pageable } from "relay-cursor-paging"
+import { artworkConnection } from "schema/v2/artwork"
+import { ResolverContext } from "types/graphql"
+import { createPageCursors } from "../fields/pagination"
+
+export const newWorksFromGalleriesYouFollow: GraphQLFieldConfig<
+  {},
+  ResolverContext
+> = {
+  type: artworkConnection.connectionType,
+  args: pageable({}),
+  description: "A list of artworks from galleries the user follows.",
+  resolve: async ({}, args, { followedProfilesArtworksLoader }) => {
+    if (!followedProfilesArtworksLoader) return null
```

Comment:
I think we should consistently throw warnings here. So we can easily diagnose crashes, unauthenticated access attempts etc.

Reply:
Hmm, I'm not sure we really need to throw a warning. I have a feeling this check is more to silence TypeScript. These authenticated loaders (like `followedProfilesArtworksLoader`) should always be there if the auth token is valid and the user is authenticated.

Returning `null` is the consistent behavior in the codebase right now. We could look together to see if it would be a good idea to raise an error here. What do you think?

Reply:
Perhaps an error is thrown elsewhere when the query tries to access fields without an authenticated user. It would be interesting to find out.

Reply:
we do that I believe in a middleware somewhere I believe. Let me check it out

Reply:
I think it's here https://github.com/artsy/metaphysics/blob/e18262f651dc35419bf317e0e5c5e9f4f9c3ddc0/src/schema/v2/test/utils.ts